python/tests/conftest.py

```diff
@@ -303,6 +303,30 @@ def mistralai_unit_test_env(monkeypatch, exclude_list, override_env_param_dict):
     return env_vars
 
 
+@fixture()
```

Comment:
nit: You can actually create a new `conftest.py` file under the `onnx/` directory to avoid making this file longer.